library/std/src/os/fd/owned.rs

```diff
@@ -355,3 +355,34 @@ impl From<OwnedFd> for crate::net::UdpSocket {
         ))))
     }
 }
+
+#[stable(feature = "io_safety", since = "1.63.0")]
```

Comment:
I think this actually landed for 1.64.0.

Reply:
Not sure where you see it in that milestone. Triagebot shows it in 1.64.0.

@cuviper do you have time to make a follow up pr? not sure when I'll find time

Reply:
`io_safety` is a different feature; it was added to 1.63 in https://github.com/rust-lang/rust/pull/95118#event-6813166988.

Reply:
Fixed in #99523.